server/data/resources/cheque-requests.json

```diff
@@ -0,0 +1,117 @@
+{
+  "title": "Cheque Request Forms",
+  "formLinks": [
+    {
+      "title": "Club Cheque Request Form",
+      "link": "https://mathsoc.uwaterloo.ca/wp-content/uploads/Club-Cheque-Request-Form.pdf"
+    },
+    {
+      "title": "MathSoc Cheque Request Form",
+      "link": "https://mathsoc.uwaterloo.ca/wp-content/uploads/MathSoc-Cheque-Request-Form-1.pdf"

```

Comment:
We'll need to deal with moving these files to the rebuilt site before we migrate to this new version.  Not only will we need to have them on our own server, but we'll also need to give admins the ability to upload updated versions of the cheque request forms.

Could you create an issue to build an interface for updating these forms?  We don't necessarily need it as part of this PR, but it will be important to figure out soon.

Reply:
Good catch - I didn't notice that about the urls. Here's the issue I made: https://github.com/MathSoc/mathsoc-website/issues/136

Feel free to edit it, of course

Reply:
🙏